File: tests/mgen-rust/stmt_continue.expected.rs

fn main() {
    let mut sum: i64 = 0_i64;
    {
        let mut i: i64 = 0_i64;
        while (i < 10_i64) {
            if (i == 5_i64) {
                continue;}
            sum += i;
            { i += 1; i };
        }
    }
    return sum;
}

File: tests/mgen-min-c/stmt_continue.expected.c
#include <stdlib.h>
#include <string.h>
#include <stdio.h>
#include <stdbool.h>
#include <stdint.h>
#include <limits.h>
#include "sn_minimal.h"

int main() {
    long long __sn__sum = 0LL;
    {
        for (long long __sn__i = 0LL; sn_lt_long(__sn__i, 10LL); __sn__i++) {
            if ((__sn__i == 5LL)) {
                continue;
            }
            __sn__sum = __sn__sum + __sn__i;
        }
    }
    sn_assert((__sn__sum == 40LL), "expected sum to be 40 (skipping 5)");
    return 0;
}
